### DISCUSSION TITLE
Placeholder for Upgrade Cost

### DISCUSSION THREAD

**Discussion by Superalex0102** [BG-Software-LLC/SuperiorSkyblock2#135] [Pending Feature Requests] [answered]
Instead of writing the full price eachtime in the upgrade menu file, simply add a placeholder to show the amount in different ways, either in formatted (e.g 90M) or with commas (90,000,000)
```
          - '&7'
          - '&dNext Level: &e2'
          - '&7'
          - '&7Purchasing the hopper upgrade'
          - '&7will increase your island''s'
          - '&7max hoppers limit.'
          - '&7'
          - '&dAmount: &f16x Hoppers'
          - '&dPrice: &f$1,000,000'
          - '&7'
          - '&aLeft-click to purchase upgrade.'
```

After the update:
```
          - '&7'
          - '&dNext Level: &e2'
          - '&7'
          - '&7Purchasing the hopper upgrade'
          - '&7will increase your island''s'
          - '&7max hoppers limit.'
          - '&7'
          - '&dAmount: &f16x Hoppers'
          - '&dPrice: &f$%price%'
          - '&7'
          - '&aLeft-click to purchase upgrade.'
```

**Comment by OmerBenGera** [comment]
I don't think it's that necessary, as the current way you can format the price however you want.
The price is fixed, therefore a placeholder wasn't added. Furthermore, checking for this placeholder everytime seems to be an unnecessary task for me, at least.
I am against this feature, but if it gets upvotes, I'll add it :)

**Reply by Superalex0102** [reply]
I'm just asking because I have a file of 2900 lines with 72 upgrades and because each has a "has-next-level" and a "no-next-level" I will have to write it 144 times and it just takes too much unnecessary time. The placeholder would be optional, if someone wants to save time they will use it, if not they just do the same task.

**Comment by Lemvs** [comment, marked as answer]
This option is implemented from 2022 😅:
- {0} - raw cost
- {1} - formatted cost (with commas)
- {2} - formatted cost (with shortcuts like 1000 = 1K)

@OmerBenGera you can make this answered and close it.
